### PR TITLE
Adding an extra message to the migrate result

### PR DESCRIPTION
**Pull request by livarcocc**
Adding an extra message to the migrate result to direct people to a place where they can get help for their migrated projects.

@piotroko @piotrpMSFT @jonsequitur @krwq @jgoshi @blackdwarf @DustinCampbell

### CODE REVIEW

File: src/dotnet/commands/dotnet-migrate/LocalizableStrings.cs [hunk 1]
```diff
@@ -30,5 +30,7 @@ internal class LocalizableStrings
         public const string CmdSkipBackupDescription = "Skip moving project.json, global.json, and *.xproj to a `backup` directory after successful migration.";
 
         public const string MigrationFailedError = "Migration failed.";
+
+        public const string MigrationAdditionalHelp = "Your project has been migrated to .csproj format and can be used with this build of .NET Core Tools.  Please visit https://aka.ms/coremigration to report issues or for help.";
```
Comment (DustinCampbell) [inline src/dotnet/commands/dotnet-migrate/LocalizableStrings.cs]: Reads to me like there's a few words missing.

"Your project has been migrated to **the** .csproj format and can be used with this build of **the** .NET Core Tools.  Please visit https://aka.ms/coremigration to report issues or **ask** for help."

Comment (livarcocc) [inline src/dotnet/commands/dotnet-migrate/LocalizableStrings.cs]: @blackdwarf Are you ok with these changes?

Comment (blackdwarf) [inline src/dotnet/commands/dotnet-migrate/LocalizableStrings.cs]: Sharp eyes @DustinCampbell :) Yeah, add the articles and the verb. :)

**Comment by livarcocc** [comment]
@dotnet-bot Test Debian8.2 x64 Debug Build

**Comment by livarcocc** [comment]
@MattGertz for approval.

**Comment by MattGertz** [comment]
Agreed.  That will help with the ones that we don't get to for the next release.

**Comment by livarcocc** [comment]
@dotnet-bot Test Fedora23 x64 Debug Build

**Pull request merged**